.github/workflows/container.yml

```diff
@@ -0,0 +1,56 @@
+name: Container
```

Comment:
Hi @da2ce7, I would change the name to something like:

`Generate local docker image`

Or any other alternative, but I think `Container` is too generic.

Why do you create an independent workflow if the putout (the local docker image) is only used in the testing workflow?

I think you can simply move these steps to the `integration` job in the `testing` workflow for now.